code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1185,7 +1185,9 @@
 	..()
 
 /datum/reagent/stimulum/on_mob_life(mob/living/carbon/M)
+	var/tox_damage = current_cycle*0.1*REM // 1 toxin damage per cycle at cycle 10
 	M.adjustStaminaLoss(-2*REM, 0)
+	M.adjustToxLoss(tox_damage, 0)
```

Comment:
just do the calc in here, If you were pulling from perf the var used to save it from pulling `current_cycle` multiple times